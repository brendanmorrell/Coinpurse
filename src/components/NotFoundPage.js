import React from 'react';

const NotFoundPage = () => (
  <div>
    <h2>Error 404! Page not found!</h2>
    {/* <Link to='/'>Go Home</Link> Link is a component that makes a client side routed link instead of causing a full-page refresh, which would happen if you used a normal link or typed the address into the browser */}
  </div>
);

export default NotFoundPage;
