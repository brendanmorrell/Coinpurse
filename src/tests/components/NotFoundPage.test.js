import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../components/NotFoundPage';

test('should display an h2 with an error 404 message', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
