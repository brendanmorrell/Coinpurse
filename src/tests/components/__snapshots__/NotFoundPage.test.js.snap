// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display an h2 with an error 404 message 1`] = `
<div>
  <h2>
    Error 404! Page not found!
  </h2>
</div>
`;
